diff --git a/config.lzma-windows.h b/config.lzma-windows.h
index c0442e5..745b013 100644
--- a/config.lzma-windows.h
+++ b/config.lzma-windows.h
@@ -155,7 +155,12 @@
 /* #undef HAVE_ICONV */
 
 /* Define to 1 if you have the <immintrin.h> header file. */
-#define HAVE_IMMINTRIN_H 1
+#if defined(__i386__) || defined(__x86_64__) \
+		|| defined(_M_IX86) || defined(_M_X64)
+#	define HAVE_IMMINTRIN_H 1
+#else
+/* #undef HAVE_IMMINTRIN_H */
+#endif
 
 /* Define to 1 if you have the <inttypes.h> header file. */
 #define HAVE_INTTYPES_H 1
@@ -294,7 +299,12 @@
 /* #undef HAVE__FUTIME */
 
 /* Define to 1 if _mm_movemask_epi8 is available. */
-#define HAVE__MM_MOVEMASK_EPI8 1
+#if defined(__i386__) || defined(__x86_64__) \
+		|| defined(_M_IX86) || defined(_M_X64)
+#	define HAVE__MM_MOVEMASK_EPI8 1
+#else
+/* #undef HAVE__MM_MOVEMASK_EPI8 */
+#endif
 
 /* Define to the sub-directory where libtool stores uninstalled libraries. */
 #define LT_OBJDIR ".libs/"
